lib/actions/map.js

```diff
@@ -40,8 +40,8 @@ export function clearLocation (payload) {
 export function zoomToStop (stop) {
   return function (dispatch, getState) {
     if (!stop) return
+    dispatch(setMapZoom({ zoom: 17 }))
     dispatch(setMapCenter({ lat: stop.lat, lon: stop.lon }))
```

Comment:
When zoomed out, these actions seemed to be called, but no UI changes happen in response.

Reply:
As far as I can tell this is a Leaflet bug where if you take any user action on the map the map updates don't work. The same bug can be seen when planning a trip, going to the route viewer (letting the map pan to the route) and then going back to the itinerary view. This will work, so long as the map wasn't dragged while in the route viewer.

I'm not sure if this behavior was ever intentional.

Reply:
Noted in #499.